libcni/api_test.go

```diff
@@ -122,11 +123,13 @@ func newPluginInfo(configValue, prevResult string, injectDebugFilePath bool, res
 }
 
 func resultCacheFilePath(cacheDirPath, netName string, rt *libcni.RuntimeConf) string {
-	return filepath.Join(cacheDirPath, "results", netName+"-"+rt.ContainerID+"-"+rt.IfName)
+	fName := fmt.Sprintf(netName + "-" + rt.ContainerID + "-" + rt.IfName)
+	return filepath.Join(cacheDirPath, "results", fmt.Sprintf("%q", fName))
```

Comment:
`%q` in this place does not do any escaping - it's adding enclosing quotation which leads to construct an output in form like: `C:\Users\appveyor\AppData\Local\Temp\1\cni_cachedir523654598\config\"apitest-some-container-id-some-eth0"`

Maybe usage of https://github.com/kennygrant/sanitize/blob/master/sanitize.go#L169 would be better there, on whole string produced by `filepath.Join`?